internal/operators/odf/validations.go

```diff
@@ -133,30 +96,23 @@ func (o *operator) computeNodeResourceUtil(cluster *models.Cluster, host *models
 		odfClusterResources.numberOfDisks += diskCount
 		odfClusterResources.hostsWithDisks++
 	}
+
 	return status, nil
 }
 
 // used to validate resource requirements for ODF
-func (o *operator) canODFBeDeployed(hosts []*models.Host, odfClusterResources *odfClusterResourcesInfo) (bool, string) {
+func (o *operator) canODFBeDeployed(
+	odfClusterResources *odfClusterResourcesInfo,
+	mode odfDeploymentMode,
+) (bool, string) {
 	var status string
```

Comment:
nit: we can remove status variable and just directly return the output of `o.setStatusInsufficientResources` and `fmt.Sprintf` functions.